data/user/articad.md

```diff
@@ -0,0 +1,8 @@
+---
+url: 'https://www.articad.com'
+image: articad.png
+name: Articad Ltd Design to Help You Sell
+added: 2019-03-25T19:57:46.494Z
+---
+This is where you fill in how you use Ember
```

Comment:
```suggestion
Design to Help You Sell
```

or another description of your choice re: how you use Ember

Reply:
https://github.com/ember-learn/ember-website/blob/master/data/user/intercom.md they just wrote `Intercom's main web application is built with Ember.js`

Reply:
```suggestion
ArtiCAD uses ember.js for it's room planner and styler application. 
```

Reply:
Thanks for your pointers